packages/core/src/index.ts

```diff
@@ -27,7 +27,7 @@ import { andThen, Result, showError } from "utils/Error";
 import { prettyPrintFn } from "utils/OtherUtils";
 import { bBoxDims, toHex } from "utils/Util";
 import { Canvas } from "renderer/ShapeDef";
-import { showOps } from "synthesis/Mutation";
+import { showMutations } from "synthesis/Mutation";
```

Comment:
```
index.ts:4:3 - error TS2305: Module '"../../node_modules/@penrose/core/build/dist"' has no exported member 'showMutations'.

4   showMutations,
```
I'm getting this error message here, so I cannot check the existing example functionality at the moment. I'm not quite sure if it has anything to do with changes made on another branch or not though.